src/components/PageComponents/Config/Position.tsx

```diff
@@ -74,8 +74,8 @@ export const Position = () => {
               name: "positionFlags",
               value: activeFlags,
               isChecked: (name: string) =>
-                activeFlags?.includes(name as FlagName) ?? false,
-              onValueChange: onPositonFlagChange,
+              activeFlags?.includes(name as FlagName) ?? false,
+\             onValueChange: onPositonFlagChange,
```

Comment:
```suggestion
              onValueChange: onPositonFlagChange,
```

Invalid char throwing error on page load: 
![image](https://github.com/user-attachments/assets/fba1e5d3-175f-4065-bca1-625e53e68b89)